code-reviews.md

```diff
@@ -0,0 +1,141 @@
+# Code Review Guide
+
+> A friendly guide for reviewing code--and not each other--at CFPB.
```

Comment:
I'm a super-friendly guy (or trying to be these days)

![Dave Dancing](https://gimmebar-assets.s3.amazonaws.com/5514a4536b6f2.gif)